DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Delegates/AllDelegates/SearchableDelegateViewModel.cs

```diff
@@ -13,7 +13,8 @@ public class SearchableDelegateViewModel : BaseFilterableViewModel
         public SearchableDelegateViewModel(
             DelegateUserCard delegateUser,
             IEnumerable<CustomFieldViewModel> customFields,
-            IEnumerable<CustomPrompt> promptsWithOptions
+            IEnumerable<CustomPrompt> promptsWithOptions,
+            int page

```

Comment:
We discussed this in the peer review: https://github.com/TechnologyEnhancedLearning/DLSV2/pull/881#discussion_r795670315
It's called page here because this is the view model for the card that appears in a list, so Page is referring to the page of the list the card appears on.